### DISCUSSION TITLE
Documentation for SEB Version Restrictions in config tool

### DISCUSSION THREAD

**Discussion by Spruce8913** [SafeExamBrowser/seb-win-refactoring#955] [Q&A]
**Is your feature request related to a problem? Please describe.**
I noticed new options were added in the "Security" tab of the SEB configuration (Windows):
![image](https://github.com/user-attachments/assets/339e4c46-3abc-4cd5-b7d5-92cf081b61f9)
It was unclear to me how I could enforce minimal SEB requirements on both mac and Windows users from just one .seb file.

**Describe the solution you'd like**
Would it be possible to extend [the documentation](https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane) online?

**Describe alternatives you've considered**
Or can I just list "Win.3.4.min" and something like "Mac.3.3.min" under each other?

**Comment by dbuechel** [comment]
> Would it be possible to extend [the documentation](https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane) online?

That we should definitely do, but unfortunately we have very limited capacity at the time being, so we can't promise to be able to update the documentation in the near future.

> Or can I just list "Win.3.4.min" and something like "Mac.3.3.min" under each other?

Exactly, that's how it works. In your case you could thus simply enter the following:

```
Win.3.4.min
Mac.3.3.min
```

**Comment by Spruce8913** [comment]
This configuration wasn't working for me (I could still open the SEB file in older version despite the restriction).
Is it possible this configuration setting is ignored in SEB versions where it didn't exist yet? (like SEB-windows 3.3 or others before SEB-windows 3.6.0 when it was implemented according to the release notes)?

If so, does this mean this setting cannot practically be used yet until years later when most used SEB versions can understand this setting?

**Reply by Spruce8913** [reply]
Also, is this also available on mac? I don't see it in [the release notes](https://github.com/SafeExamBrowser/seb-mac/releases).

**Reply by danschlet** [reply]
You‘re completely right, this is not yet available on Mac, it hopefully will make it into version 3.4. And yes, this feature is quite useless now. I would think in a year it can be used, as we remove older versions (at least Mac) from the download repositories 

This is why version checks and restrictions should be implemented in the assessment systems or LMS. Some systems with SEB integrations built in such features. Other didn‘t, although we always recommended that to anyone who asked us about it.

**Reply by Spruce8913** [reply]
Thank you for the very fast response! If I may be so bold: it may be wise to document this feature already including the warning that this feature doesn't work yet on (still present) SEB-versions. Other teachers like me may decide to use this feature, without knowing about its current practical restrictions.

**Reply by hribeir0** [reply]
Hi all,
Thank you very much. I, too, was trying this feature and scratching my head as to why it isn't being enforced even when I was experimenting with Win 3.5.0
Thanks once again for your great work!
